code/src/components/Summary.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+const Summary = ({ nameInput, value, whatTeam, watchedStatus }) => {
+
+    return (
+        <section className="end-screen">
+            <h2>Your Buffy profile:</h2>
+            <div className="stats-container">
+            <p>Rating: {value}</p>
+
+            <p>Name: {nameInput}</p>
+
+            <p> Shipping: {whatTeam === 'Angel' ? 'Team Angel' : 'Team Spike'}</p>
+
+
+            <p>Seen it: {watchedStatus}</p>
+            </div>
+
+            <div className='print-btn'><button onClick={window.print}>Print your results!</button></div>
```

Comment:
I cant see this button for some reason on the page

Reply:
My bad - wanted to make like a "profile card" for printing as a fun thing, using the @media print -query function, but didn't have time to finish it so i just disabled all of it.